principles/architectural.md

```diff
@@ -5,4 +5,22 @@ title:  "Architectual Principles"
 
 # Architectual Practices
 
-Placeholder
+## Introduction
+These pages give advice on the common approaches that we take across the Apprenticeship service. It will detail what approaches should be taken and why.
+
+The approaches and patterns listed are expected to be followed but are not aimed at limiting new components, aspects or approaches. They are given to show the currently preferred approach within the service. Where a new approach or type of implementation is required for a project then it is perfectly acceptable that the approaches listed can be updated. If a new approach is to be taken by a project team then agreement and acceptance within the architecture group needs to be sought and agreed before any design is finalised (No implementation should be carried out without this).  
+
+Our overall approach is driven from the GDS service design manual ([service standard](https://www.gov.uk/service-manual/service-standard)). 
+
+Some example of these are
+* Iterate quickly
+* Support incremental change
+
+Also from the [Technology Code of Practice](https://www.gov.uk/government/publications/technology-code-of-practice/technology-code-of-practice)
+* Cloud first
+* Open Standards
+
+### Supported Architecture Styles (sub-pages todo)
```

Comment:
I think the list below will potentially confuse people without the supporting pages.  Many would argue that EDA and APIs are ways to implement a micro-service architecture etc.  Just wondering if it may generate more questions than it answers if details not present under it.

Reply:
I agree but need to get something out there!

Reply:
Next task is to add those pages